src/mimefactory.rs

```diff
@@ -507,6 +507,39 @@ impl<'a> MimeFactory<'a> {
             .unprotected
             .push(Header::new("MIME-Version".into(), "1.0".into()));
 
+        headers
+            .unprotected
+            .push(Header::new_with_value("From".into(), vec![from]).unwrap());
```

Comment:
So headers are in order they are pushed?  why not move the mime-version down as well as we're tiding this up?  i think it is customary to start with from, to etc.

Reply:
I have rearranged all headers now, the order is:
1. Standard headers
2. Auto-Submitted
3. Non-standard (Autocrypt, Chat-*)
4. MIME (Mime-Version, ideally also content-type but it's placed separately so I didn't check where it goes)

Reply:
:+1: that's a good order